app/proto/conversations.proto

```diff
@@ -108,16 +110,18 @@ enum HostRequestStatus {
   HOST_REQUEST_STATUS_REJECTED = 2;
   HOST_REQUEST_STATUS_CONFIRMED = 3;
   HOST_REQUEST_STATUS_CANCELLED = 4;
+  HOST_REQUEST_STATUS_MAYBE_ACCEPTED = 5;
```

Comment:
What is maybe accepted?

Reply:
If the host says "maybe" - I thought HOST_REQUEST_STATUS_MAYBE isn't clear if the host or surfer is saying maybe

Reply:
I didn't know hosts could say maybe. Seems weird to me. Was this a requested feature? In any event, code looks good.

Reply:
Yes, both from design and people on the forum